ICAFIX/hcp_fix_multi_run

```diff
@@ -176,14 +189,48 @@ then
 else
 	this_script_dir=$(dirname "$0")
 fi
+  
+# determine if legacy fix or pyfix will be used
+legacyFixEnabled=$(opts_StringToBool "$legacyFixEnabled")
+if ((legacyFixEnabled)) && [[ -f ${FSL_FIXDIR}/fix ]] && [[ ! -f ${FSL_FIXDIR}/convert_fix_model ]];then
+	legacyFix=1
+	log_Msg "Using legacy fix from ${FSL_FIXDIR}"
+	else
+	if [[ ! -f ${FSL_FIXDIR}/convert_fix_model ]];then
```

Comment:
Guard against unset variable (since the new `SetUp` comments it out).
```suggestion
	if [[ "${FSL_FIXDIR:-}" == "" || ! -f "$FSL_FIXDIR"/convert_fix_model ]];then
```